tests/e2e_test.go

```diff
@@ -44,8 +45,13 @@ func init() {
 
 var bundleJsonApi *httptest.Server
 
-// Reset the RPC endpoint and mock backend servers
-func resetTestServers() {
+// Setup RPC endpoint and mock backend servers
+func testSetup() {
+	db := database.NewMockStore()
+	serverSetup(db)
+}
```

Comment:
why split this into 2 functions?

Reply:
Because depends on the test, we need to pass in mem store or mock store..If you have someother ideas here, let me know.I can update it accordingly.

Reply:
maybe rename `testSetup` to `serverSetupWithMockStore`